securedrop/translations/he/LC_MESSAGES/messages.po

```diff
@@ -972,7 +960,7 @@ msgid "Additional Action Required"
 msgstr "נדרשת פעולה נוספת"
 
 msgid "Log Back In"
-msgstr ""
+msgstr "כנס בחזרה‫"
```

Comment:
flagging that there's a hidden bidirectional indicator here (U+202b) - not an issue for 2.12.0 unless/until Hebrew is included.